modules/openapi-generator/src/test/resources/2_0/petstore-with-fake-endpoints-models-and-additional-properties-for-testing.yaml

```diff
@@ -0,0 +1,2009 @@
+swagger: '2.0'
```

Comment:
Why make this separate yaml file?
To test disallowAdditionalPropertiesIfNotPresent=True/False?
If that is the case, can you make this spec much smaller, give it a file name like:
`issues_1369_1371.yaml` or
`additional_properties_spec.yaml`
and use it in DefaultCodegenTest.java?
We want to minimize the spec additions that we make.

Reply:
> Why make this separate yaml file?
> To test disallowAdditionalPropertiesIfNotPresent=True/False?
> If that is the case, can you make this spec much smaller, give it a file name like:
> `issues_1369_1371.yaml` or
> `additional_properties_spec.yaml`
> and use it in DefaultCodegenTest.java?
> We want to minimize the spec additions that we make.

Because initially I modified the existing PR, but it caused a huge number of changes because there are many other languages using the original file. Sure I can make this file smaller.

Reply:
I reduced the size of the file.